src/sentry/options/manager.py

```diff
@@ -77,7 +77,8 @@ def __init__(self, store):
         self.store = store
         self.registry = {}
 
-    def set(self, key, value, coerce=True):
+    # TODO: Find out why `from sentry.models.options import OptionsTypes` doesn't work at top level
```

Comment:
My guess is cyclic imports, or Django getting mad that you're using models before the application is 'ready'. For the cyclic imports you can hack around that using:

```python
if typing.TYPE_CHECKING:
   from sentry.models.options.option import OptionsType
```

And then defining the type hints as strings.

Reply:
Or `from __future__ import annotations` and you can skip the strings

Reply:
Cool. I'll give those a shot. Thanks!